docs/content/docs/dev/table/sqlClient.md

```diff
@@ -91,6 +91,40 @@ The `SET` command allows you to tune the job execution and the sql client behavi
 After a query is defined, it can be submitted to the cluster as a long-running, detached Flink job.
 The [configuration section](#configuration) explains how to declare table sources for reading data, how to declare table sinks for writing data, and how to configure other table program properties.
 
+### Key-strokes
+
+There is a list of available key-strokes in SQL client
+
+| Key-Stroke (Linux, Windows(WSL)) | Key-Stroke (Mac) | Description                                |
+|:---------------------------------|------------------|:-------------------------------------------|
+| `alt-b`                          | `Esc-b`          | Backward word                              |
+| `alt-f`                          | `Esc-f`          | Forward word                               |
+| `alt-c`                          | `Esc-c`          | Capitalize word                            |
+| `alt-l`                          | `Esc-l`          | Lowercase word                             |
+| `alt-u`                          | `Esc-u`          | Uppercase word                             |
+| `alt-d`                          | `Esc-d`          | Kill word                                  |
+| `alt-n`                          | `Esc-n`          | History search forward                     |
+| `alt-p`                          | `Esc-p`          | History search backward                    |
```

Comment:
it's not 100% same.
`history search forward/backward` behaves same as `Up/line from history` in case there is no input provided.
However if there is a non-empty input it will navigate to the first occurrence `forward/backward` containing input.
That's the difference

Reply:
i added this clarification